Objective-C/LogSink/CBLLogTypes.h

```diff
@@ -0,0 +1,54 @@
+//
+//  CBLLogTypes.h
+//  CouchbaseLite
+//
+//  Copyright (c) 2024 Couchbase, Inc All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ Log domain.
+ */
+typedef NS_OPTIONS(NSUInteger, CBLLogDomain) {
+    kCBLLogDomainAll            = 1 << 0 | 1 << 1 | 1 << 2 | 1 << 3 | 1 << 4, ///< All domains
```

Comment:
Would this be better as
kCBLLogDomainDatabase | kCBLLogDomainQuery | ...

Reply:
I actually did that for Swift as the namings are way shorter. As you now mentioned it now, I will change that for readability for Obj-C as well. Thanks!